usaspending_api/agency/v2/views/federal_account_list.py

```diff
@@ -20,15 +23,20 @@ class FederalAccountList(PaginationMixin, AgencyBase):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.params_to_validate = ["fiscal_year", "filter"]
+        self.params_to_validate = ["fiscal_year", "filter", "bureau_slug"]
+
+    def format_results(self, file_a_response, file_b_response):
```

Comment:
I know this isn't originally your code, but many of the steps in this final `format_results` function could use some inline comments walking the code-reader through the logic and flow of what's happening with the data-reshaping and accumulation.